src/mcpserver.ts

```
@@ -11,5 +11,22 @@ const server = new McpServer({
     },
 });
 
+server.registerTool(
+    "get_current_datetime",
+    {
+        title: "Get current datetime",
+        description: "Get the current systemtime and date",
+        inputSchema: {
+        },
+    },
+    async ({ }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)